Symfony/CS/Fixer/Contrib/PhpUnitStrictFixer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+final class PhpUnitStrictFixer extends AbstractFixer
+{
+    static private $phpUnitMethods = array(
+        "assertAttributeEquals" => "assertAttributeSame",
+        "assertAttributeNotEquals" => "assertAttributeNotSame",
+        "assertEquals" => "assertSame",
+        "assertNotEquals" => "assertNotSame",
```

Comment:
I often use `assertSame` and `assertNotEquals` since I often want a less strict not equals check. Maybe we should move that not equals change to a different fixer?